src/sgidClient.ts

```diff
@@ -43,12 +43,17 @@ export class SgidClient {
     hostname = 'https://api.id.gov.sg',
   }: SgidClientParams) {
     // TODO: Discover sgID issuer metadata via .well-known endpoint
+    const issuer =
+      (API_VERSION as number) === 1
+        ? new URL(hostname).origin
+        : `${new URL(hostname).origin}/v${API_VERSION}`
```

Comment:
super nit: What do you think of extracting `new URL(hostname).origin` out from the ternary operation? ie.

`const issuer = new URL(hostname).origin + (API_VERSION as number) === 1 ? '' : `/v${API_VERSION)`

Reply:
hmm i think it's a good choice to refactor - however, i decided to do it in another way to be more explicit that only version 2 onwards includes the suffix

```javascript
    let issuer = new URL(hostname).origin
    if ((API_VERSION as number) !== 1) {
      issuer += `/v${API_VERSION}`
    }
```

feel free to let me know what you think!